packages/perspective-node/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "perspective-node",
```

Comment:
It provides canary builds for testing packages on CI. If we’re not doing a lot of new dev, we can remove it.